ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/yum/tasks/main.yml

```diff
@@ -252,3 +253,9 @@
   tags:
     - yum
     - adoptopenjdk
+
```

Comment:
Any reason not to just invoke it as `gpg` when it's used here instead of using a symlink?

Reply:
The package_signature_verification.sh invokes it as gpg so id rather keep it consistent. The other versions of the package, the latest one for example, installs it with the symlink. Version 2.0.30 (the working version) keeps it as `/usr/bin/gpg2` oddly

Reply:
I'd be a bit more tempted to adjust the shell script to handle both cases instead of having an untracked symbolic link in the system binaries directory. What would happen in the future if we tried to update to the latest version and this symlink was still in place (bearing in mind it wouldn't get removed along with the current version of the package)? Does this have the potential to cause is problems with future playbook deployments?

I won't block on the approach, however if we're going to stick with the symlink please add a comment above the section with an explanation even if it's just pointing to your comment e.g. `For why this symlink is necessary, see  https://github.com/adoptium/infrastructure/pull/3020#discussion_r1162721897`

Reply:
> What would happen in the future if we tried to update to the latest version and this symlink was still in place (bearing in mind it wouldn't get removed along with the current version of the package)? Does this have the potential to cause is problems with future playbook deployments?

It is similar to other package instalments such as `ant` where we manually make a `/usr/bin/ant` link https://github.com/adoptium/infrastructure/blob/558b23d2cc1436f658593b2d10dc444a88fb031a/ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/ant/tasks/main.yml#L28. 

In the case of a future update, I would suggest a general rule where we observe how we installed the former package via the playbook, and then undo any manual work if necessary before performing the update.

I'll add the comment